CHANGELOG.md

```diff
@@ -5,6 +5,48 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.3.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.3.0) - 2023-09-25
+
+This release provides support for the Spend before Sync synchronization
+strategy, and contains the following updates:
+
+### Security
+
+- Fix database concurrency bugs that could have led to panics or incorrect history tree data (#7590)
+
+### Added
+
+- Add missing elasticsearch flag feature to lib docs (#7568)
+- Add missing Docker variables and examples (#7552)
+
+#### Spend before Sync support
+
+- Add state requests and support code for the `z_getsubtreesbyindex` RPC (#7408)
+- Implement the `z_getsubtreesbyindex` RPC (#7436)
+- Add snapshot tests for the `z_getsubtreesbyindex` RPC (#7514)
+- Test the `z_getsubtreesbyindex` RPC (#7515, #7521, #7566)
+- Format subtree roots in little-endian order (#7466)
+- Add note subtree indexes for new and existing blocks (#7437)
+- Upgrade subtrees from the tip backwards, for compatibility with wallet syncing (#7531)
+- Handle a subtree comparison edge case correctly (#7587)
+
+### Changed
+
+- Check database format is valid every 5 minutes, to catch format errors in new block code (#7602)
+- Check database format is valid on shutdown, to catch format errors in new block code (#7606)
```

Comment:
```suggestion
- Check database format is valid every 5 minutes and on shutdown, to catch format errors in new block code (#7602, #7606)
```

Reply:
We just reverted one of these changes in PR #7627, so I'd suggest instead:
```suggestion
- Check database format is valid on shutdown in production, to catch rare database edge-cases (#7606). We expect to catch almost all of these errors in CI (#7602, #7627).
```